common-content/en/module/tools/read-about-operating-systems/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Operating systems"
+headless = true
+time = 120
+facilitation = false
+emoji= "💻"
+objectives = [
+    "Define an operating system.",
+    "Describe what an kernel is.",
+    "Explain what a process is.",
+    "List what processes are created when running `ls | grep '[A-Z]'`.",
+    "Explain what a system call (syscall) is.",
+    "Give three examples of syscalls.",
+]
++++
+
+{{<note type="Reading">}}
+Read chapter 10 of How Computers Really Work.
+
+Do every exercise listed in the chapters.
+
+You only need to do the projects listed below (though are welcome to try any others that you want!)
+
+Check you have achieved each learning objective listed on this page.
+{{</note>}}
+
+{{<note type="Project">}}
+Do project 23 from How Computers Really Work.
+
+You can do this on any Unix OS - you do not need a Raspberry Pi.
```

Comment:
Do trainees know what this is? Worth tooltipping to explain that Linux and macOS are both flavours of Unix?